lib/spec/util/extensions/miq-uri_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "spec_helper"
+
+$:.push(File.expand_path(File.join(File.dirname(__FILE__), %w{.. .. .. util extensions})))
```

Comment:
You shouldn't need this.  `require 'util/miq-uri'` should be sufficient for this spec